code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -83,6 +83,42 @@
 		M.satiety += 30
 	. = ..()
 
+/datum/reagent/consumable/cooking_oil
+	name = "Cooking Oil"
+	id = "cooking_oil"
+	description = "A variety of cooking oil derived from fat or plants. Used in food preparation and frying."
+	color = "#EADD6B" //RGB: 234, 221, 107 (based off of canola oil)
+	taste_mult = 0.8
+	taste_description = "oil"
+	nutriment_factor = 7 * REAGENTS_METABOLISM //Not very healthy on its own
+	metabolization_rate = 10 * REAGENTS_METABOLISM
+	var/fry_temperature = 450 //Around ~350 F (117 C) which deep fryers operate around in the real world
+	var/boiling //Used in mob life to determine if the oil kills, and only on touch application
+
+/datum/reagent/consumable/cooking_oil/reaction_obj(obj/O, reac_volume)
+	if(holder && holder.chem_temp >= fry_temperature)
+		if(isitem(O))
+			O.loc.visible_message("<span class='warning'>[O] rapidly fries as it's splashed with hot oil! Somehow.</span>")
+			var/obj/item/reagent_containers/food/snacks/deepfryholder/F = new(get_turf(O))
+			F.fry(O, volume)
+
+/datum/reagent/consumable/cooking_oil/reaction_mob(mob/living/M, method = TOUCH, reac_volume, show_message = 1, touch_protection = 0)
+	if(!istype(M))
+		return
+	if(holder && holder.chem_temp >= fry_temperature)
+		boiling = TRUE
+	if(method in list(VAPOR, TOUCH)) //Directly coats the mob, and doesn't go into their bloodstream
+		if(boiling)
+			M.visible_message("<span class='warning'>The boiling oil sizzles as it covers [M]!</span>", \
+			"<span class='userdanger'>You're covered in boiling oil!</span>")
+			M.emote("scream")
+			playsound(M, 'sound/machines/fryer/deep_fryer_emerge.ogg', 25, TRUE)
+			var/oil_damage = (holder.chem_temp / fry_temperature) * 2 //Damage taken per unit
+			M.adjustFireLoss(min(60, oil_damage * reac_volume)) //Damage caps at 60
```

Comment:
You only need about 14 units heated to 1000 degrees kelvin to hit the damage cap of 60.
That means you only need 28 units of heated cooking oil to put someone in crit.
That's pretty broken.

Reply:
I'll mess with a number or two.

Reply:
Microwaves instantly heat to 1000 too, don't they?